packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -152,7 +159,21 @@ function WorklistDataGrid(props) {
 
   const getCellData = (cell, cellColumnIndex) => {
     const tabIndex = { tabIndex: '-1' };
-    return props.rowHeaderIndex === cellColumnIndex ? (<th key={cellColumnIndex} {...tabIndex} className={cx('worklist-data-grid-row-header')}>{cell.content}</th>) : (<td key={cellColumnIndex} {...tabIndex} className={cx('worklist-data-grid-cell-data')}>{cell.content}</td>);
+
+    // Determine whether cell is a header or grid cell
+    const WorklistCellTag = props.rowHeaderIndex === cellColumnIndex ? 'th' : 'td';
+
+    return (
+      // Return worklist data grid cell component
+      <WorklistCellTag
+        key={cellColumnIndex}
+        {...tabIndex}
+        className={cx('worklist-data-grid-cell', { masked: cell.isMasked })}
+        aria-label={cell.isMasked ? intl.formatMessage({ id: 'Terra.worklistDataGrid.maskedCell' }) : undefined}
```

Comment:
The AC mentioned that users should be able to define a reason for the masking but since the grid doesn't convey a reason visually, I think we can leave it like this. (No change required, just making a note.)